xml/dhcp_vendor_class.xml

```diff
@@ -367,6 +367,52 @@
     <param pos="0" name="os.vendor" value="Cisco"/>
   </fingerprint>
 
+  <fingerprint pattern="^CISCO SPA8800$">
+    <description>Cisco SPA8800 VoIP Gateway</description>
+    <example>CISCO SPA8800</example>
+    <param pos="0" name="hw.vendor" value="Cisco"/>
+    <param pos="0" name="hw.device" value="VoIP"/>
+    <param pos="0" name="hw.product" value="SPA8800 IP Telephony Gateway"/>
+    <param pos="0" name="hw.cpe23" value="cpe:/h:cisco:spa8800_ip_telephony_gateway:-"/>
+    <param pos="0" name="os.vendor" value="Cisco"/>
+    <param pos="0" name="os.product" value="SPA8800 8-Port IP Telephony Gateway Firmware"/>
+    <param pos="0" name="os.cpe23" value="cpe:/o:cisco:spa8800_8-port_ip_telephony_gateway_firmware:-"/>
+  </fingerprint>
+
+  <fingerprint pattern="^CISCO SPA112$">
+    <description>Cisco SPA112</description>
```

Comment:
Shouldn't this be a bit more enlightening about what the device actually is? (IIUC, ATA stands for Analog Telephone Adapter)

Reply:
I took a quick look an existing SIP fingerprint for the SPA112 and it didn't have an expansion of SPA, however, it can be helpful to provide additional details in the description. These details could be added in a follow-up PR. From some quick searching it appears SPA might be a "shared port adapter". Although, in this case the name of the [SPA112](https://www.cisco.com/c/en/us/products/unified-communications/spa112-2-port-phone-adapter/index.html) appears to be "Cisco SPA112 2-Port Phone Adapter" and makes no reference to "shared port adapter" and only "port adapter".

Reply:
Sorry Dan, didn't see this till after the PR was merged.
Yes, as Matthew identified, there is an existing fingerprint for the SPA112 device in sip_banners, so I used the same description. 2 fingerprints for a single device with different descriptions looked confusing to me.